api/region.yaml

```diff
@@ -255,6 +255,24 @@ functions:
     @param {Marionette.View} view
     @param {Object} options
 
+  renderView: |
+    Utility function to allow overriding of logic that determines whether the shown view should `render`. It will
+    always `render` by default.
+    
+    @api public
+    @param {Marionette.View} view
+    @param {Object} options
+    @returns {void}
+
+  shouldDestroyView: |
+    Utility function to allow overriding of logic that determines whether the old view should be destroyed. It will
+    always `destroy` by default.
```

Comment:
Should this say 'always returns true' or something to that effect instead? Might complicate things, but should we mention `preventDestroy` will affect the default value too?